scripts/deploy.sh

```diff
@@ -0,0 +1,74 @@
+export DEBIAN_FRONTEND=noninteractive
+
+# Install build dependencies
+echo "deb http://ftp.de.debian.org/debian sid main" >> /etc/apt/sources.list
+apt update -y
+apt install git curl ufw gcc make nginx -y
+apt install gcc-10 -y
```

Comment:
I know this is a actual dependency, but if some day you get to understand why we need gcc to run a already compiled binary, I would be glad to understand 😬

Reply:
Hahaha, I was wondering the same, but was hoping on you guys having an answer to my question! Isn't there a dynamic library that is provided by GCC and used by MeiliSearch or one of the Rust crates?

Reply:
🤷‍♀️